client/lib/terms/test/common/index.js

```diff
@@ -7,8 +7,8 @@ var Dispatcher = require( 'dispatcher' ),
 /**
  * Internal dependencies
  */
-var data = require( './data' ),
-	ActionTypes = require( '../constants' ).action;
+var data = require( '../data' ),
```

Comment:
I moved non-test files to subfolders. This is something we should do anyway before we drop `tests.json` files.